packages/zipkin/index.d.ts

```diff
@@ -19,6 +19,7 @@ declare namespace zipkin {
     id: TraceId;
 
     scoped<V>(callback: () => V): V;
+    local<V>(name: string, callback: () => V): V;
```

Comment:
@pbadenski what should we put here to represent a callback that returns a `Promise<V>`? would it be `local<V>(name: string, callback: () => Promise<V>): V;` if so, is there an import needed? or do we declare it locally..